aea/multiplexer.py

```diff
@@ -380,7 +383,11 @@ async def _disconnect_all(self) -> None:
         self.logger.debug("Tear the multiplexer connections down.")
         for connection_id, connection in self._id_to_connection.items():
             try:
-                await asyncio.wait_for(self._disconnect_one(connection_id), timeout=5)
+                await asyncio.wait_for(
+                    self._disconnect_one(connection_id), timeout=self.DISCONNECT_TIMEOUT
+                )
+            except FuturesTimeoutError:
```

Comment:
A timeout can be caused if one of the connections (E.g. server) takes too long. This should be muted as it can be "normal" behaviour

Reply:
@solarw

Reply:
@DavidMinarschshould discuss and set policy for timeouts on every connection method.